htdocs/fourn/commande/card.php

```diff
@@ -857,8 +857,19 @@
 		}
 
 		if (!$error) {
-			$db->commit();
+			// reopen order if necessary
+			if ($object->status == CommandeFournisseur::STATUS_RECEIVED_COMPLETELY) {
+				if ($object->setStatus($user, CommandeFournisseur::STATUS_RECEIVED_PARTIALLY) < 0) {
+					$db->rollback();
```

Comment:
Making a rollback here means we will make duplicate rollback (because it is also done few lines later.

Reply:
Thanks @eldy ; it is fixed now.